Berksfile

```diff
@@ -0,0 +1,8 @@
+source "https://supermarket.getchef.com"
+
+metadata
+
+# Solr
+cookbook 'solr',
+    :git => 'git://github.com/inviqa/chef-solr.git',
+    :ref => 'master'
```

Comment:
also `source "https://supermarket.getchef.com"` now `source "https://supermarket.chef.io"` (tho it redirects)